v2/visualrecognitionv4/visual_recognition_v4.go

```diff
@@ -1811,6 +1842,12 @@ func (options *CreateCollectionOptions) SetDescription(description string) *Crea
 	return options
 }
 
+// SetTrainingStatus : Allow user to set TrainingStatus
+func (options *CreateCollectionOptions) SetTrainingStatus(trainingStatus *TrainingStatus) *CreateCollectionOptions {
+	options.TrainingStatus = trainingStatus
+	return options
+}
+
```

Comment:
are we allowing the user to set training status here?

Reply:
@kevinkowa good catch - I think the hand edit that removed this got edited out. I'll remove this

Reply:
updated